<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comand</class>
 <widget class="QWidget" name="comand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>На главную</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectAll">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Показать все отделы</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Общий Email отдела</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Название компании заказчика</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnUpdate">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="leComandEmail"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Название отдела</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="leComandName"/>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="leCompanyName"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Отделы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="twComand">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leComandNameSearch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string extracomment="Введите ФИО Ученика для поиска"/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Окно отладки</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teComand">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
